README.md

```diff
@@ -71,7 +71,7 @@ To self-host the Box Content Preview library, follow these steps:
 1. Either fork the repo and check out the version you want to host or download the specific version as a zip:
   * Check out a specific version with `git checkout v2.8.0`
   * Download a specific version as a zip from https://github.com/box/box-content-preview/releases
-2. Install dependencies and build the library with `yarn install && yarn prod`
+2. Install dependencies and build the library with `yarn install && yarn build && yarn:build:prod`
```

Comment:
build prod is missing the i18n step so I added regular build as well.

Reply:
Can we use `yarn build:i18n` somewhere in there instead of doing `yarn build`?